### PR TITLE
Add support for IAB Track Files (SMPTE ST 2067-201)

### PR DESCRIPTION
**Pull request by palemieux**
See test at https://github.com/sandflow/iab-in-imf

### CODE REVIEW

File: src/AS_02_IAB.cpp [hunk 1]
```diff
@@ -0,0 +1,617 @@
...
+
+#include "AS_02_internal.h"
+#include "AS_02_IAB.h"
+
+#include <iostream>
+#include <iomanip>
+#include <array>
+
+namespace Kumu {
+  class RuntimeError : public std::runtime_error {
```
Comment (IMFTool) [inline src/AS_02_IAB.cpp:38]: On macOS Catalina, clang-1100.0.33.17, the build fails here:
```
/Users/wolfgangruppel/src/asdcplib-sandflow/asdcplib-sandflow/src/AS_02_IAB.cpp:38:9: error: exception specification of overriding function is more lax than base version
  class RuntimeError : public std::runtime_error {
        ^
```

Adding `-std=c++11` as compiler option solves the problem, but enforcing C++11 might cause issues on other platforms.

Comment (palemieux) [reply src/AS_02_IAB.cpp:38]: The objective is compatibility with C++98, so I plan to fix it.

What about setting `set_property(TARGET ${PROJECT_NAME} PROPERTY CXX_STANDARD 98)` in `src/CMakeList.txt`?

Comment (palemieux) [reply src/AS_02_IAB.cpp:37]: @IMFTool Addressed at https://github.com/cinecert/asdcplib/pull/33/commits/8be2ad09b94fbf2f023e24cf1b7eec10c03047b4

Would you like to propose a change to `CMakeList.txt`?

Comment (IMFTool) [reply src/AS_02_IAB.cpp:37]: That fixes the compiler error. No need to Change `CMakeLists.txt` IMHO.

Comment (palemieux) [reply src/AS_02_IAB.cpp:37]: @IMFTool The error is triggered on Ubuntu and Win10 only when `CMakeLists.txt` explicitly constrain the compiler to C++98.

Comment (IMFTool) [reply src/AS_02_IAB.cpp:37]: By default, clang compiles according to C++98 [1], while Linux gcc and VS by default compile according do C++ >=11.
With the [latest patch](https://github.com/cinecert/asdcplib/commit/8be2ad09b94fbf2f023e24cf1b7eec10c03047b4), the PR build fine on macOS with default clang setting (i.e. C++98).
[1] [https://clang.llvm.org/cxx_status.html](https://clang.llvm.org/cxx_status.html)

**Comment by jhursty** [comment]
Thanks for this Pierre. Everyone else please review and comment (or let me know that you have no comment.) Review will close May 7.

File: src/AS_02_IAB.cpp (hunk 1 above)
Comment (msheby) [reply src/AS_02_IAB.cpp:37]: Tossing a 
```#include <stdexcept>```
up in src/AS_02_IAB.cpp fixed that same build issue for me on one of my CentOS boxes. It built fine as-is on a Fedora one.

Comment (palemieux) [reply src/AS_02_IAB.cpp:38]: Addressed at https://github.com/cinecert/asdcplib/pull/33#discussion_r421689866

**Comment by jhursty** [comment]
accepting into this branch for further development

**Pull request merged**